openedx/core/djangoapps/appsembler/settings/settings/aws_common.py

```diff
@@ -40,7 +40,22 @@ def plugin_settings(settings):
             "MANDRILL_API_KEY": settings.MANDRILL_API_KEY,
         }
         settings.INSTALLED_APPS += ("anymail",)
-
+    # Mandrill Subaccount Support
+    settings.MANDRILL_SUBACCOUNT = settings.AUTH_TOKENS.get("MANDRILL_SUBACCOUNT")
```

Comment:
nitpick: It seems like a regular ID and not a secret, right?

If so, please use `ENV_TOKENS` instead.

```suggestion
    settings.MANDRILL_SUBACCOUNT = settings.ENV_TOKENS.get("MANDRILL_SUBACCOUNT")
```

Reply:
@OmarIthawi Good point, @amirtds in case you decided to do the change, you need to do it on edX configs as well.

Reply:
Good point @OmarIthawi Makes sense. I move it to `ENV_TOKENS`. Thanks both